java/midp/com/sun/midp/security/Permissions.java

```diff
@@ -1,1230 +0,0 @@
-/*
```

Comment:
We've decided to *git mv* the files in java/custom/ instead of copying them in #1234 (there's a discussion in the issue to explain the reasons).